source/common/network/address_impl.h

```diff
@@ -84,6 +86,7 @@ class Ipv4Instance : public InstanceBase {
   bool operator==(const Instance& rhs) const override;
   const Ip* ip() const override { return &ip_; }
   const Pipe* pipe() const override { return nullptr; }
+  const EnvoyInternalAddress* envoyInternalAddress() const override { return nullptr; }
```

Comment:
we're aiming for 97% coverage in Envoy, so even little no-op functions like this should get unit tests please.

Reply:
I remember I add this check to cover this line in unit test. Checking...